erc20-migration/hardhat.config.js

```diff
@@ -16,6 +16,10 @@ else {
     mnemonic: (process.env.MNEMONIC || "")
   }
 }
+
+let priorityFeeInWei = process.env.PRIORITY_FEE || "100"
```

Comment:
here and below: let's use the old method (no overrides) if both env  variables are not present.